dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaAPIHostsTask.java

```diff
@@ -66,19 +63,19 @@ protected Void doRun(
         JsonNode jsonHosts;
         try (CloseableHttpResponse hostsResponse =
             httpClient.execute(new HttpGet(hostPerClusterUrl))) {
-          jsonHosts = objectMapper.readTree(hostsResponse.getEntity().getContent());
+          jsonHosts = getObjectMapper().readTree(hostsResponse.getEntity().getContent());
         }
         String stringifiedHosts = jsonHosts.toString();
         writer.write(stringifiedHosts);
         writer.write('\n');
 
-        ApiHostListDTO apiHosts = objectMapper.readValue(stringifiedHosts, ApiHostListDTO.class);
+        ApiHostListDTO apiHosts =
+            getObjectMapper().readValue(stringifiedHosts, ApiHostListDTO.class);
```

Comment:
`getObjectMapper` called twice, you can assign variable to it and use it here.

Reply:
As I can see `getObjectMapper` returns a predefined object so it should not be any extra calculation or memory allocation here.

Reply:
It's valid point. In my opinion both approaches are valid, I will keep the method call because it's utility object which less important in main flow.